woodwork/tests/type_system/test_ltype_inference.py

```diff
@@ -2,32 +2,30 @@
 import woodwork as ww
 from woodwork.logical_types import (
     Boolean,
+    BooleanNullable,
     Categorical,
     Datetime,
     Double,
     Integer,
+    IntegerNullable,
     LogicalType,
     NaturalLanguage,
     Timedelta
 )
+from woodwork.tests.testing_utils import to_pandas
 from woodwork.utils import import_or_none
 
 UNSUPPORTED_KOALAS_DTYPES = [
     'int32',
-    'Int64',
     'intp',
     'uint8',
     'uint16',
     'uint32',
     'uint64',
     'uintp',
-    'float32',  # compatible starting with 1.4.0
     'float_',
     'object',
-    'datetime64[ns]',  # compatible starting with 1.4.0
     'category',
-    'string',
-    'boolean',
 ]
```

Comment:
This never got updated when Koalas added support for these dtypes and WW changing the limit on the min version of Koalas.